cloud/benefits.mdx

```diff
@@ -1,29 +1,63 @@
 ---
 title: Benefits
-description: Discover the benefits of Flipt Enterprise Cloud and why it's the best choice for your organization
+description: Discover the benefits of Flipt Managed Cloud and why it's the best choice for your organization
 mode: "wide"
 ---
 
-At Flipt, we aim to provide the best possible feature flagging experience for our users. That's why we're excited to introduce Flipt Enterprise Cloud, a new offering that combines the best of both worlds: the simplicity and flexibility of open-source Flipt with the scalability and reliability of a managed cloud service.
+<Info>
+  Flipt Managed Cloud is currently in beta. Some functionality may be subject to
+  change.
 
-## We Hate Limits
+Interested in joining the beta? [Request access here](https://flipt.cloud).
 
-One of the biggest challenges with traditional feature flagging solutions is that they often come with limits. Whether it's the number of flags you can create, the number of users you can have, or the amount of data you can store, these limits can be frustrating and restrictive. With Flipt Enterprise Cloud, we've removed these limits, giving you the freedom to create as many flags as you need, invite as many users as you want, and store as much data as you require.
+</Info>
 
-We believe that you should never have to worry about hitting a limit or being charged extra for using too much of a service. With Flipt Enterprise Cloud, you can focus on building great products without having to worry about artificial constraints.
+At Flipt, we aim to provide the best possible feature flagging experience for our users. That's why we're excited to introduce Flipt Managed Cloud, a new offering that combines the best of both worlds: the simplicity and flexibility of open-source Flipt with the scalability and reliability of a managed cloud service.
```

Comment:
I almost want to swap those last two benefits around, but maybe I am thinking about it wrong.
Managed service is about simplicity and prebuilt integrations. Self-hosting is about control of scale, availability, security and cost.
```suggestion
At Flipt, we aim to provide the best possible feature flagging experience for our users. That's why we're excited to introduce Flipt Managed Cloud, a new offering that combines the best of both worlds: the simplicity and flexibility of a managed cloud service with the scalability and reliability of self-hosted Flipt.
```